x/incentive/keeper/keeper.go

```diff
@@ -41,3 +41,8 @@ func (k Keeper) ModuleBalance(ctx sdk.Context, denom string) sdk.Coin {
 	amount := k.bankKeeper.SpendableCoins(ctx, authtypes.NewModuleAddress(incentive.ModuleName)).AmountOf(denom)
 	return sdk.NewCoin(denom, amount)
 }
+
```

Comment:
would be good to use `Keeper` only for data access, and implement the logic in the msg_server or another struct.